### DISCUSSION TITLE
History of clicked links

### DISCUSSION THREAD

**Discussion by taras-zak** [will-stone/browserosaurus#491] [Q&A] [answered]
Hi, found this [issue](https://github.com/will-stone/browserosaurus/issues/149), that resolved  in https://github.com/will-stone/browserosaurus/releases/tag/v10.0.0.
>Browserosaurus will now keep a small history of clicked links

Where I can found such history ? Or its limited to "Restore recently closed URL" feature?

**Comment by will-stone** [comment, marked as answer]
The full history was removed for security reasons: https://github.com/will-stone/browserosaurus/releases/tag/v10.3.0 

Browserosaurus now only keeps the last url selected for as long as it’s running.
